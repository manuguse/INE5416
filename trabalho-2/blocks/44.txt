1 1 7 7 7 11 11 19 19 19
1 7 7 8 11 11 12 19 19 19
1 1 8 8 12 12 12 20 19 21
2 1 8 8 13 12 12 12 21 21
3 3 3 3 13 13 16 15 22 22
3 5 3 9 14 14 16 15 15 22
3 5 9 9 14 15 15 15 22 22
4 5 5 9 14 15 17 17 18 22
5 5 9 9 14 14 14 18 18 22
6 5 9 10 10 10 18 18 18 18