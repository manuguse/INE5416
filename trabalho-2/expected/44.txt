5 1 4 5 1 4 1 4 7 2
4 3 2 3 2 3 7 3 6 1
2 6 5 2 1 4 6 1 5 3
1 3 4 1 3 2 5 3 2 1
4 6 7 5 2 1 2 7 6 7
3 7 1 6 7 4 1 6 4 5
2 6 2 5 6 2 3 5 1 4
1 5 1 3 5 1 2 1 5 3
2 4 7 1 3 2 1 2 4 2
1 3 4 3 2 1 7 1 3 6